llvm/lib/Target/AArch64/GISel/AArch64CallLowering.cpp

```diff
@@ -53,6 +53,8 @@
 using namespace llvm;
 using namespace AArch64GISelUtils;
 
+extern cl::opt<bool> EnableSVEGISel;
```

Comment:
This shouldn't need to spread. I don't like having fallBackToDAGISel at all, but I see we have 2 of them. Why not just put them both in the same place?

Reply:
`AArch64CallLowering::fallBackToDAGISel` works on `MachineFunction` as an early path before IRTranslate. 
`AArch64TargetLowering::fallBackToDAGISel` works on each `Instruction`.

FastISel also uses `AArch64TargetLowering::fallBackToDAGISel`, making it more complex to merge the logic.

This patch does not completely support SVE formal arguments, especially for predicate registers <vscale x 16 x i1>. 
The option is needed until we support them.